code/datums/ai/basic_mobs/basic_ai_behaviors/run_away_from_target.dm

```diff
@@ -16,6 +18,8 @@
 
 /datum/ai_behavior/run_away_from_target/perform(seconds_per_tick, datum/ai_controller/controller, target_key, hiding_location_key)
 	. = ..()
+	if (!controller.blackboard[BB_BASIC_MOB_FLEEING])
```

Comment:
i forgot, is fleeing a boolean or does it resolve to the mob it's fleeing from? use `isnull()` if it's the latter but i could just be smallbrained rn

Reply:
It's just a boolean, don't worry I was thinking of you.